### PR TITLE
[Maid-3107] chore/ci: update scripts to Rust stable 1.28.0 

### PR DESCRIPTION
**Pull request by m-cat**
nightly to 2018-07-07
clippy to 0.0.212
rustfmt to 0.99.2

### CODE REVIEW

File: src/nat/util.rs [hunk 1]
```diff
@@ -38,13 +38,13 @@ pub fn enable_so_reuseport(_sock: &TcpBuilder) -> io::Result<()> {
 /// A replacement for `IpAddr::is_global` while we wait for that to enter stable.
 pub fn ip_addr_is_global(ip: &IpAddr) -> bool {
     match *ip {
-        IpAddr::V4(ref addr_v4) => ipv4_addr_is_global(addr_v4),
```
Comment (pierrechevalier83) [inline src/nat/util.rs]: I would probably put these in a separate commit (about making latest clippy happy) like the rustfmt change, but yymv

Comment (m-cat) [reply src/nat/util.rs]: Done!
